xbmc/pvr/PVRManager.cpp

```diff
@@ -861,85 +860,17 @@ void CPVRManager::UpdateCurrentChannel(void)
       !IsPlayingChannel(m_currentFile->GetPVRChannelInfoTag()))
   {
     m_currentFile.reset(new CFileItem(playingChannel));
-    UpdateItem(*m_currentFile);
+    UpdateCurrentFile();
     m_bIsChannelPreview = false;
   }
 }
 
 void CPVRManager::UpdateCurrentFile(void)
 {
   CSingleLock lock(m_critSection);
-  if (m_currentFile)
-    UpdateItem(*m_currentFile);
-}
-
-bool CPVRManager::UpdateItem(CFileItem& item)
-{
-  /* Don't update if a recording is played */
-  if (item.IsPVRRecording())
-    return false;
-
-  if (!item.IsPVRChannel())
-  {
-    CLog::Log(LOGERROR, "CPVRManager - %s - no channel tag provided", __FUNCTION__);
-    return false;
-  }
-
-  CSingleLock lock(m_critSection);
-  if (!m_currentFile || !m_currentFile->GetPVRChannelInfoTag() || !item.GetPVRChannelInfoTag() ||
-      *m_currentFile->GetPVRChannelInfoTag() == *item.GetPVRChannelInfoTag())
-    return false;
-
-  if (!m_bIsChannelPreview)
-    g_application.SetCurrentFileItem(*m_currentFile);
-  
-  g_infoManager.SetCurrentItem(m_currentFile);
 
-  CPVRChannelPtr channelTag(item.GetPVRChannelInfoTag());
-  CPVREpgInfoTagPtr epgTagNow(channelTag->GetEPGNow());
-
-  if (channelTag->IsRadio())
-  {
-    CMusicInfoTag* musictag = item.GetMusicInfoTag();
-    if (musictag)
-    {
-      musictag->SetTitle(epgTagNow
```

Comment:
This is dead code since you removed the call to CPVRManager::UpdateItem by no longer calling CInputStreamPVRMananager::UpdateItem.